src/main/resources/db/migration/V012__remove_message_id_constraint.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE notifications
```

Comment:
not 100% sure about this, may need to discuss with you.

Reply:
Makes sense, its not providing us a lot of value currently anyway and will obviously be removed upon us switching to this new API so all good. 

Will need to remember during testing to run this script against demo just so we covered it off